src/components/App/App.js

```diff
@@ -36,7 +37,7 @@ const App = () => {
 
   return (
     <main>
-      <Link to="/" style={{textDecoration: 'none'}}><Header /></Link>
+      <Header />
```

Comment:
Any reason why you removed the link here? I might need it back for the edit profile picture view